source/features/parse-backticks.tsx

```diff
@@ -18,7 +18,8 @@ features.add({
 	screenshot: 'https://user-images.githubusercontent.com/170270/55060505-31179b00-50a4-11e9-99a9-c3691ba38d66.png',
 	include: [
 		features.isDiscussionList,
-		features.isCommitList
+		features.isCommitList,
```

Comment:
I think isCommitList is already included in isRepo, but I wonder if isRepo is too broad. `include` should be as specific as possible

Reply:
You're right, I'll update this :)

Reply:
See 51fdb91

Reply:
I think you’re looking for isRepoTree